lib/widgets/resources/arm_widgets/pose_widget.dart

```diff
@@ -11,6 +14,9 @@ class PoseWidget extends StatefulWidget {
   State<PoseWidget> createState() => _PoseWidgetState();
 }
 
+bool _isLive = false;
+bool _isGoingToPose = false;
```

Comment:
Do these need to be global? It doesn't look to me like the places they're being used are outside the `_PoseWidgedState` class where `_isLive` was defined before.